flocker/node/_container.py

```diff
@@ -713,5 +716,7 @@ def calculate_changes(self, desired_configuration, current_cluster_state,
         if start_restart:
             phases.append(in_parallel(changes=start_restart))
 
-        return sequentially(changes=phases,
-                            sleep_when_empty=timedelta(seconds=1))
+        return sequentially(
+            changes=phases,
+            sleep_when_empty=NOOP_SLEEP_TIME
```

Comment:
I'm not sure about our coding guidelines, but usually when I move to this formatting I also do a trailing comma, so future diffs will only add/remove lines there are interested in.